clab/common.sh

```
@@ -51,8 +51,22 @@ load_local_image_to_kind() {
+    # archive whose index points at the whole multi-platform manifest list,
+    # while only the platform we pulled has its blobs present locally. "kind
+    # load image-archive" then runs "ctr images import --all-platforms", which
+    # tries to resolve every referenced manifest and fails with
+    #   ctr: content digest sha256:...: not found
+    # Restricting the archive to the platform we actually pulled keeps the
+    # index single-manifest and imports cleanly.
+    if [[ $CONTAINER_ENGINE == "docker" ]] && docker save --help 2>&1 | grep -q -- '--platform'; then
+        save_opts=(--platform "${PLATFORM}")
+    fi
+
     rm -f ${temp_file}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)